src/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -145,6 +145,9 @@ internal ZipArchiveEntry(ZipArchive archive, string entryName)
         /// </summary>
         public ZipArchive Archive => _archive;
 
+        [CLSCompliant(false)]
```

Comment:
There are other uint properties in this library that don't have CLSCompliant. Do we need to bother with it.

Reply:
I believe so, based on this comment in the original PR: https://github.com/dotnet/corefx/pull/25204#discussion_r150416742